lib/osf-components/addon/components/addons-service/manager/component.ts

```diff
@@ -106,9 +138,11 @@ export default class AddonsServiceManagerComponent extends Component<Args> {
     @waitFor
     async getAddonProviders() {
         const serviceStorageProviders: Provider[] = await taskFor(this.serviceStorageProviders).perform();
-
-        serviceStorageProviders.sort(this.providerSorter);
-        this.addonProviders = serviceStorageProviders;
+        this.storageProviders = serviceStorageProviders;
+        // Get other provider types here, eventually
+        const allProviders = [...serviceStorageProviders];
+        allProviders.sort(this.providerSorter);
+        this.allAddonProviders = allProviders;
```

Comment:
We won't be getting all providers, so this seems unnecessary, yes? It'll always be filtered providers?

Reply:
Oh, yup! Forgot about this leftover bit!